### ISSUE TITLE
Customer-reported issue with NNBD migration tool: Could not load edit details

### ISSUE THREAD

**Issue by mnordine** [dart-lang/sdk#42117]
Could not load edit details

Error: Error reaching migration preview server.

Please fill in the following:

**Name of package being migrated (if public)**: StageXL
**What I was doing when this issue occurred**: I clicked on highlighted question mark to get more info on why it was made nullable
**Is it possible to work around this issue**:
**Has this issue happened before, and if so, how often**: Every time I click a question mark
**Dart SDK version**: 2.9.0-edge.83277a4f37
**Additional details**: Windows

Thanks for filing!

Stacktrace: _auto populated by migration preview tool._

```
Error reaching migration preview server.
    at Object.b (http://localhost:49634/C:/Users/Mark/code/dart/StageXL?authToken=qH6Wfh5EZR4%3D:257:3)
    at http://localhost:49634/C:/Users/Mark/code/dart/StageXL?authToken=qH6Wfh5EZR4%3D:3042:9
    at Gs.a (http://localhost:49634/C:/Users/Mark/code/dart/StageXL?authToken=qH6Wfh5EZR4%3D:1424:72)
    at Gs.$2 (http://localhost:49634/C:/Users/Mark/code/dart/StageXL?authToken=qH6Wfh5EZR4%3D:4336:23)
    at SX.$2 (http://localhost:49634/C:/Users/Mark/code/dart/StageXL?authToken=qH6Wfh5EZR4%3D:4331:25)
    at Ji.rp (http://localhost:49634/C:/Users/Mark/code/dart/StageXL?authToken=qH6Wfh5EZR4%3D:4605:25)
    at Fe.Kw (http://localhost:49634/C:/Users/Mark/code/dart/StageXL?authToken=qH6Wfh5EZR4%3D:4388:27)
    at RW.$0 (http://localhost:49634/C:/Users/Mark/code/dart/StageXL?authToken=qH6Wfh5EZR4%3D:4536:7)
    at Object.HZ (http://localhost:49634/C:/Users/Mark/code/dart/StageXL?authToken=qH6Wfh5EZR4%3D:1469:80)
    at vs.ZL (http://localhost:49634/C:/Users/Mark/code/dart/StageXL?authToken=qH6Wfh5EZR4%3D:4451:3)
```

**Comment by srawlins** [comment]
CC @stereotype441 @jcollins-g @MichaelRFairhurst

**Comment by jcollins-g** [comment]
@stereotype441 this looks like number 4 from the email.

**Comment by stereotype441** [comment]
We believe this is a known issue with Windows support.  Once https://github.com/dart-lang/sdk/issues/40381 is fixed we should look into whether this is still happening.

**Comment by stereotype441** [comment]
On further reflection, I'm going to suggest the opposite: let's fix this issue, and then once it's fixed re-evaluate #40381 to see if any other WIndows issues remain :)

**Comment by mnordine** [comment]
Confirmed the fix, works on my end, :)
